incubator/virtualcluster/pkg/syncer/conversion/helper.go

```diff
@@ -227,3 +222,19 @@ func BuildVirtualPersistentVolume(cluster, vcNS, vcName string, pPV *v1.Persiste
 	vPV.Spec.ClaimRef.UID = vPVC.UID
 	return vPV
 }
+
+// IsControlPlaneService will return if the namespacedName matches the proper
+// NamespacedName in the tenant control plane
+func IsControlPlaneService(service *v1.Service, cluster string) bool {
+	kubernetesNamespace := ToSuperMasterNamespace(cluster, metav1.NamespaceDefault)
+	kubernetesServices := "kubernetes"
```

Comment:
Can we change kubernetesServices to KubernetesService?

Reply:
you want to export that? the previous vars weren't exported.

Reply:
Done! 👍